### DISCUSSION TITLE
Deployment not directing to "/" and comes with 404 NOT FOUND error

### DISCUSSION THREAD

**Discussion by shriram-muthukrishnan** [vercel/community#4959] [Help]
### Summary

I tried following the steps for deploying my flask python project on vercel. 
Inspite of adding vercel.json to the folder, I am still getting the 404 NOT FOUND and home page is not displaying.
Any pointers on this would be appreciated.

### Steps to Reproduce

Deployment succeeds
Web browser display 404 NOT FOUND for my movie recommendation project

**Comment by amyegan** [comment]
Hi @shriram-muthukrishnan. Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that we can use to help with debugging?
